cli/src/commands/git/clone.rs

```diff
@@ -115,15 +117,24 @@ pub fn cmd_git_clone(
     // `/some/path/.`
     let canonical_wc_path = dunce::canonicalize(&wc_path)
         .map_err(|err| user_error_with_message(format!("Failed to create {wc_path_str}"), err))?;
-    let clone_result = do_git_clone(
+
+    let (workspace, repo, git_repo) = setup_workspace(command, args.colocate, &canonical_wc_path)?;
+    writeln!(
+        ui.status(),
+        r#"Fetching into new repo in "{}""#,
+        canonical_wc_path.display()
+    )?;
```

Comment:
nit: I think this can be moved to `fetch_new_repo()`. The path should be identical to `workspace_command.workspace_root()`.